src/Npgsql.NetTopologySuite/PostGisGeometryType.cs

```diff
@@ -0,0 +1,45 @@
+namespace Npgsql.NetTopologySuite
+{
+    /// <summary>
+    /// PostGIS Geometry types as defined in the OGC WKB Spec
+    /// </summary>
+    internal enum PostGisGeometryType
+    {
+
+        /// <summary>
+        /// The OGIS geometry type number for points.
+        /// </summary>
+        Point = 1,
```

Comment:
Any cost/benefit to including a `None = 0`?

Reply:
I honestly don't know, copy-paste :)